src/frontend/src/components/ChangeRequestDetailCard.tsx

```diff
@@ -110,7 +110,7 @@ const ChangeRequestDetailCard: React.FC<ChangeRequestDetailCardProps> = ({ chang
   const ChangeRequestTypeView = () => determineChangeRequestTypeView(changeRequest);
   const pillColor = determineChangeRequestPillColor(changeRequest.type);
   return (
-    <Card sx={{ maxWidth: 300 }}>
+    <Card sx={{ width: 300, mr: 1, mb: 1 }}>
```

Comment:
can you add  borderRadius: 5 here to make it more consistent with the project details card